package.json

```diff
@@ -22,7 +22,8 @@
   "exports": {
     ".": {
       "node": "./src/sign.wat.js",
-      "default": "./dist/sign.wat.js"
+      "default": "./dist/sign.wat.js",
+      "types": "./index.d.ts"
```

Comment:
IIRC, it's somehow important that the types property in "exports" comes first (before node and default)

Reply:
Locally that worked for me even with types on last position, but more than happy to move it up.